lib/jsonschema.lua

```diff
@@ -120,6 +120,19 @@ function codectx_mt:localvar(init, nres)
   return unpack(names)
 end
 
+function codectx_mt:localvartab(init, nres)
+    local names = {}
+    local nloc = self._nloc
+    nres = nres or 1
+    for i=1, nres do
+      names[i] = sformat('locals.var_%d_%d', self._idx, nloc+i)
```

Comment:
style: `nloc+i` -> `nloc + i`

Reply:
@membphis fixed the style problems.
And the same style problems in old localvar are also fixed:)